### PR TITLE
Cb 5 team component

### PR DESCRIPTION
**Pull request by vishnubapana**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds a team component to introduce all the member of the team.

* **What is the current behavior? What does the code do before you made the new changes?** 
Previously didn't have a team component, now there is one.

* **What is the new behavior?**
Displays cards that show team members.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No breaking change is present

* **Other information**:

### CODE REVIEW

File: frontend/src/components/LandingPage/Team/Team.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Card, CardTitle, CardText, CardGroup, CardBody
+} from 'reactstrap';
+
+const Team = (props) => {
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: Can you change this to a functional component please.

File: frontend/src/components/LandingPage/Team/Team.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        <Card>
+          <CardBody>
+            <CardTitle>Vishnu Bapanapalli</CardTitle>
+            <CardText align='left'>Computer Science
+            <br/>
+            <a href='https://github.com/vishnubapana'>github</a>
+            <br/>
+            <CardText align='left'>Graduated with a Bachelors in Computer Engineering and 
+                currently pursuing a Masters in Computer Science with a focus in 
+                Intelligent Systems and Data Science.</CardText>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.js]: React is giving a warning here about having a p tag be a [descendant of another p tag](https://imgur.com/a/rZ19ipv). The way to fix this is to have all of this text exist in one CardText. The same change would need to happen to the other cards as well.

File: frontend/src/components/LandingPage/Team/Team.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+  
+import { shallow } from 'enzyme';
+import { screen, render } from '@testing-library/react';
+import Team from './Team'
+import React from 'react';
+
+describe('Test Team component', () => {
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Team/Team.test.js:7]: Great job writing some tests!

**Review by Codubee — CHANGES_REQUESTED** [review]
This is fantastic Venkata! One thing I wanted to mention as well is that I dont think you are importing your component into the LandingPage.js so I couldnt see anything without doing that step first.

**Review by ryannd — COMMENTED** [review]
Looks good! There's also a CardLink component on reactstrap that you can try using for the links to github as well.

File: frontend/src/components/LandingPage/Team/Team.js (hunk 1 above)
Comment (vishnubapana) [reply frontend/src/components/LandingPage/Team/Team.js]: Updated to a functional component.

File: frontend/src/components/LandingPage/Team/Team.js (hunk 2 above)
Comment (vishnubapana) [reply frontend/src/components/LandingPage/Team/Team.js]: Updated the code to have all text in one CardText.

File: frontend/src/components/LandingPage/Team/Team.test.js (hunk 3 above)
Comment (vishnubapana) [reply frontend/src/components/LandingPage/Team/Team.test.js:7]: Thanks

**Review by ryannd — APPROVED** [review]
looks great! good job

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**